### DISCUSSION TITLE
EngRes The Gathering Tomorrow! (includes a libp2p update)

### DISCUSSION THREAD

**Discussion by dhuseby** [swift-libp2p/swift-libp2p#26] [General]
The libp2p project will be giving its update in the greater network all hands tomorrow including a comprehensive update on js-libp2p. Here’s the event page: [EngRes The Gathering · Zoom · Luma](https://lu.ma/Engres-The-Gathering-02-15-24)

If there’s any swift-libp2p updates you’d like me to include in the “community updates” section, either reply here, email me at [dave.huseby@libp2p.io](mailto:dave.huseby@libp2p.io) or ping me (Dave.Slack [libp2p community]) on the [#libp2p-implementers slack channel](https://filecoinproject.slack.com/archives/C03K82MU486)

**Comment by btoms20** [comment]
Hey @dhuseby! No updates regarding swift-libp2p. I haven't devoted much time to the project lately. Hopefully in the coming months I'll be able to ramp up development. 

Thanks for checking!
